system_tests/bigquery.py

```diff
@@ -602,3 +613,18 @@ def test_dump_table_w_public_data(self):
         dataset = Config.CLIENT.dataset(DATASET_NAME, project=PUBLIC)
         table = dataset.table(TABLE_NAME)
         self._fetch_single_page(table)
+
+    def test_create_table_nested_schema(self):
+        TABLE_NAME = 'test_table'
```

Comment:
I am aware of the copious usage throughout our codebase, but I'm saying that use is inconsistent with the actual meaning of "constant" and that we shouldn't make it worse here.